### PR TITLE
Improved dark alpha gradient

### PR DESCRIPTION
**Pull request by jepsar**

### CODE REVIEW

**Comment by mdbassit** [comment]
This looks good, thanks for the fix!
One thing though, this selector `.clr-dark .clr-field button` will have no effect since that button is outside of the color picker dialog. Could you remove it?

File: src/coloris.css [hunk 1]
```diff
@@ -383,7 +383,7 @@ input.clr-color:focus {
 .clr-dark .clr-alpha div,
 .clr-dark .clr-swatches button,
 .clr-dark .clr-preview:before,
-.clr-dark .clr-field button {
+.clr-dark.clr-field button {
```
Comment (jepsar) [inline src/coloris.css:386]: @mdbassit how about allowing dark fields by adding the `clr-dark` class to them?

Comment (mdbassit) [reply src/coloris.css:386]: Hmmm, it feels a bit messy, ideally we should use CSS custom properties (variables) to solve this, but that would require a slight rewrite of the CSS, I'd rather leave that for a later version. Once things settle a bit on my side, I'll look more into this along with your request for an inline picker.
For now, can you remove that selector, and we will get back to this later?

Comment (jepsar) [reply src/coloris.css]: Alright. Updated the PR. Yeah, I was also thinking variables would be a good idea.

**Pull request merged**